resources/js/components/Checkout/Login.vue

```diff
@@ -86,11 +86,19 @@ export default {
 
         successfulLogin() {
             if (this.checkoutLogin) {
-                this.$root.checkout.step = 2
+                this.$root.checkout.step = this.nextStep
             } else if (this.redirect) {
                 Turbo.visit(window.url(this.redirect))
             }
         },
     },
+    computed: {
+        loginStep: function () {
+            return config.checkout_steps[config.store_code]?.indexOf('login')
+        },
```

Comment:
We're using this `indexOf` construction quite a bit, maybe it's worth making a function out of it?

Could make the nextStep function become something like:

```js
return checkoutStep('login') + 1
```

And also simplify the v-ifs in the blade file below.

Reply:
You're right, that's a good idea to create a simplified function. Especially for template work.